MSAL/src/public/MSALPublicClientApplication.h

```diff
@@ -101,16 +101,13 @@
 #pragma SafariViewController Support
 
 /*!
-    Returns whether or not the URL is a response to a MSAL web authentication
-    session. Call this before passing the URL into -handleMSALResponse:
- */
-+ (BOOL)isMSALResponse:(NSURL *)response;
-
-/*!
     Pass a URL response from your application delegate's openURL handler into
     MSAL for web authnetication sessions.
+ 
+    Returns whether or not the URL is a response to a MSAL web authentication
+    session
```

Comment:
nit and not important: As a developer, I'd like to see a more specific description or the return values and behavior, e.g 
I'd better like something like "returns NO if URL was not handled, because it's not a response to a MSAL web authentication and YES if URL was handled" (obviously wording is not the best here)